docs/modules/ROOT/pages/how-tos/configuring/dynamic-labels.adoc

```diff
@@ -0,0 +1,132 @@
+= Using dynamic labels
+
+Konflux allows applying some dynamic labels to images derived from properties and params of the build pipeline. The labelling is supported directly by the `LABELS` param of the *build-container* tasks. A supporting *generate-labels* task can optionally be used to produce some dynamic labels.
+
+. <<supplying-labels-to-the-build-container-task>>
+. <<generating-dynamic-labels>>
+. <<incrementing-release-label>>
+
+[NOTE]
+====
+These can be combined and used together.
+====
```

Comment:
It would be nice to give an example of combining these together

```yaml
name: LABELS
value:
  - "$(tasks.generate-labels.results.labels[*])"
  - "short-commit=$(tasks.clone-repository.results.short-commit)"
```

Reply:
Done in e48bdce.